src/config/Options.js

```diff
@@ -2074,6 +2074,7 @@ export default class Options {
 			 * @property {Boolean} [line.connectNull=false] Set if null data point will be connected or not.<br>
 			 *  If true set, the region of null data will be connected without any data point. If false set, the region of null data will not be connected and get empty.
 			 * @property {Boolean} [line.step.type=step] Change step type for step chart.<br>
+			 * @property {Array}   [line.classes=['line1', 'line2'] If set, used to set a css class on each line.
```

Comment:
The parameter value should be set as `[line.classes]`. Within square bracket, default value(which is undefined) should be set.

Reply:
Also plz add example code for this option :)